### PR TITLE
kv: refine ErrEntryTooLarge error msg

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Refine ErrEntryTooLarge error message.

### What is changed and how it works?
change ErrEntryTooLarge msg.

### Check List 

Tests 

 
 - No code

Code changes

Related changes

### CODE REVIEW

File: kv/memdb_buffer.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func (m *memDbBuffer) Set(k Key, v []byte) error {
 		return errors.Trace(ErrCannotSetNilValue)
 	}
 	if len(k)+len(v) > m.entrySizeLimit {
-		return ErrEntryTooLarge.GenWithStack("entry too large, size: %d", len(k)+len(v))
+		return ErrEntryTooLarge.GenWithStack("entry too large, the max entry size is %d, the size of date to be inserted is %d", m.entrySizeLimit, len(k)+len(v))
```
Comment (jackysp) [inline kv/memdb_buffer.go]:
```suggestion
		return ErrEntryTooLarge.GenWithStack("entry too large, the max entry size is %d, the size of data is %d", m.entrySizeLimit, len(k)+len(v))
```
I think it is better to change the definition of `ErrEntryTooLarge`.

Comment (crazycs520) [reply kv/memdb_buffer.go]: done.

**Comment by jackysp** [comment]
```
schema_validator_test.go:100:
    c.Assert(valid, Equals, ResultUnknown)
... obtained domain.checkResult = 0
... expected domain.checkResult = 2
```
This ut failed, should we fix it in this PR, or just ignore it?

**Comment by crazycs520** [comment]
I will fix this ut in the next PR.

**Pull request merged**